docs/installing.rst

```diff
@@ -27,21 +28,51 @@ In your ``conf.py`` file:
     :ref:`supported-dependencies`
         Officially Supported versions of Python, Sphinx, and other dependencies.
 
+
 .. note::
 
    Adding this theme as an extension is what enables localization of theme
    strings in your translated output. If these strings are not translated in
    your output, either we lack the localized strings for your locale, or you
    are using an old version of the theme.
 
+   ..
+      comment about this note: it's possibly not necessary to add the theme as an extension.
+      Rather, this is an issue caused by setting html_theme_path.
+      See: https://github.com/readthedocs/readthedocs.org/pull/9654
+
+
+.. _howto_upgrade:
+
+How to upgrade
+--------------
+
+Adding ``sphinx-rtd-theme`` to your project's dependencies will make pip install the latest compatible version of the theme.
+
+If you want to test a **pre-release**, you need to be explicit about the version you specify.
+Otherwise, pip will ignore pre-releases. Add for instance ``sphinx-rtd-theme==1.1.0b3`` to test a pre-release.
+
+.. tip::
+    We recommend that you pin the version of Sphinx that your project is built with.
+    We won't release sphinx-rtd-theme without marking its compatibility with Sphinx. So if you do not pin ``sphinx-rtd-theme`` itself, you will always get the *latest compatible* release.
+    
+    More information is available in Read the Docs' documentation on :doc:`rtd:guides/reproducible-builds`.
+
+.. tip:: 
+    **Read the Docs users:** If you are upgrading a project created before October 20, 2020 (January 21, 2021 for Read the Docs for Business), it's a requirement to specify the version of ``sphinx-rtd-theme``. Otherwise, ``sphinx-rtd-theme`` is pinned to a lower version to address legacy issues.
+    More information is available in Read the Docs' documentation on :doc:`rtd:build-default-versions`.
+    We recommend checking your build logs to verify which version of ``sphinx-rtd-theme`` you are running.
```

Comment:
We shouldn't be communicating date based feature flags any more than strictly necessary. I do not give any exceptions for old projects in recommendations, *all projects should specify their dependencies*.

At this point, this tip is trying to say too much. And I'm not sure it's even helpful to the user to tell them how to find the version of the theme they have installed. We're trying to tell them how to install a *new version*, so linking them to our configuration file docs and the reproducible build docs is where I would point the reader.

In the interest of getting this PR merged, we should skip this tip and come back to this when we have the content ready to link to.

Reply:
> all projects should specify their dependencies.

That is literally what the tip box above this one already says :wink: 

Not everyone have pinned their versions, for either good or bad reasons... or no reason at all. The tip box is there so that people who aren't getting the expected version of sphinx_rtd_theme on RTD have a pointer.

With your feedback, I don't have a clear picture of what sort of guidance (tip box) they should be given. It sounds like you would want a more generic "Troubleshooting your Read the Docs build? Try these articles..." tip box. But I don't get it really, I tried adjusting this a lot.

I agree we should give up, rather than continuing this comment thread :+1: I'd be interested to see your suggestion at a later point.